### ISSUE TITLE
"vendor/" Lacks README

### ISSUE THREAD

**Issue by silabs-robin** [openhwgroup/core-v-verif#2451]
### Type
* Other.

### Steps to Reproduce
`cd vendor`.
If you don't already know what that directory is, you won't find out in that directory.
Is it documented in a core-specific external repo?
A simple readme file with one sentence would inform either were the documentation is or explain what the directory is and how to use it.
For anyone wishing to use the content of that directory, that directory itself is the most natural place to look.

**Comment by MikeOpenHWGroup** [comment]
Good point @silabs-robin!  The same could be said for `vendor_lib`.  I will work with @MarioOpenHWGroup to create a couple of helpful (?) READMEs for these directories.
